IntegrationTest/src/test/java/integration/IntegrationTest.java

```diff
@@ -22,6 +22,8 @@ public void IntegrationTest1() throws Exception {
                 new AtomicLong(0));
         ServerEngine server = new ServerEngine(config);
         server.Start();
+
+        Thread.sleep(3000000);
     }
```

Comment:
should probably change sleep time to some smaller value, once we finishes our debugging process. 50 mins seems quite long.